### ISSUE TITLE
[1.5.15 EU] Invalid data when loading models

### ISSUE THREAD

**Issue by amaury10** [FrSkyRC/ETHOS-Feedback-Community#4494]
Hello,

After updating my X18 to the last stable release, some of my models have been corrupted (or are incompatible).
I tried to restore a backup with the same result : I get a popup saying "Invalid data".

I joined my model files if anyone can understand why and how I can fix it...

Regards,

Amaury.

![20240908_105520](https://github.com/user-attachments/assets/4e91d22c-5c51-4d8d-a047-a0a319d5b174)
[models.zip](https://github.com/user-attachments/files/16921579/models.zip)

**Comment by Wolepo** [comment]
Hello,
The models are displayed in the overview. Clone the ‘damaged’ models. You should then be able to open them.

**Comment by Wolepo** [comment]
Converted in the simulator
[Modelle.zip](https://github.com/user-attachments/files/16921645/Modelle.zip)

**Comment by amaury10** [comment]
Hello Wolepo,

The models you sent me seems to be working. How did you convert them ?
Do you have any idea of what went wrong ?

Thanks a lot,

Amaury.

**Comment by Wolepo** [comment]
Deactivate the error message and do not open, but clone. Then the model usually works again. Bertrand should explain why the automatic conversion sometimes does not work after an update.
Regards 
Wolfgang

**Comment by fdapport** [comment]
How to reproduce this error:
Hardware X202
Start from Ethos 1.5.11

0) I have made  the backup from Ethos suite 1.5.7
1) I have update from 1.5.11 to 1.5.15 first, using Ethos suite 1.5.9.
2) After I have update I have open a Model01 and then came back to the original model the  was set when I have update it (model 00).
3) I have update Ethos from 1.5.14 to 1.4.15 using Ethos suite 1.5.9
4) then I have opened again the Model01 and I have invalid data.
5) I open again the model00 and was ok, as was ok the other models.
6) I have cloned Model01 to modelxx and then open modelxx that was ok, but display widget was not in modelxx.

To fix the problem I have uploaded  from the backUp from 1.5.11 Model01 and that I do not have any error msg.
